### PR TITLE
Create some defaults in appengine-web.xml

### PR DESCRIPTION
**Pull request by carise**

### CODE REVIEW

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -10,13 +10,13 @@
...
-        <env-var name="POLICY_SCANNER_EXECUTE_ON_CLOUD" value="TRUE_OR_FALSE" />
-        <env-var name="POLICY_SCANNER_DATAFLOW_TMP_BUCKET" value="YOUR_GCS_BUCKET" />
-        <env-var name="POLICY_SCANNER_GIT_REPOSITORY_URL" value="URL_OF_YOUR_GIT_REPO" />
-        <env-var name="POLICY_SCANNER_GIT_REPOSITORY_NAME" value="NAME_OF_YOUR_GIT_REPO" />
+        <env-var name="POLICY_SCANNER_ORG_ID" value="YOUR_ORG_NUMBER" />
+        <env-var name="POLICY_SCANNER_INPUT_REPOSITORY_URL" value="${app.id}.appspot.com" />
+        <env-var name="POLICY_SCANNER_SINK_URL" value="gs://${app.id}.appspot.com/OUTPUT" />
+        <env-var name="POLICY_SCANNER_EXECUTE_ON_CLOUD" value="TRUE" />
+        <env-var name="POLICY_SCANNER_DATAFLOW_TMP_BUCKET" value="${app.id}.appspot.com" />
+        <env-var name="POLICY_SCANNER_GIT_REPOSITORY_URL" value="POLICY_GIT_REPO_URL" />
```
Comment (blueandgold) [inline src/main/webapp/WEB-INF/appengine-web.xml]: nit: add "scanner"
POLICYSCANNER_GIT_REPO_URL

**Comment by blueandgold** [comment]
👍

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 2]
```diff
@@ -10,13 +10,13 @@
...
+        <env-var name="POLICY_SCANNER_GIT_REPOSITORY_NAME" value="POLICY_GIT_REPO_NAME" />
```
Comment (carise) [inline src/main/webapp/WEB-INF/appengine-web.xml]: Done

File: src/main/webapp/WEB-INF/appengine-web.xml (hunk 1 above)
Comment (carise) [inline src/main/webapp/WEB-INF/appengine-web.xml]: Done

**Pull request merged**